core/src/apps/base.py

```diff
@@ -100,6 +98,15 @@ def get_features() -> Features:
             Capability.PassphraseEntry,
         ]
 
+        # Only some models have special altcoins
+        if utils.INTERNAL_MODEL in ("T2T1",):
```

Comment:
this should probably be `not utils.MODEL_IS_T2B1`, or create a `MODEL_IS_T2T1` flag

Reply:
Done in [c07d77e](https://github.com/trezor/trezor-firmware/pull/3240/commits/c07d77e831eef3124b259191b237675f9d0db3d0)